src/test/unit_tests/jest_tests/test_nc_nsfs_account_cli.test.js

```diff
@@ -1004,6 +1070,36 @@ describe('manage nsfs cli account flow', () => {
                 const res = await exec_manage_cli(type, ACTIONS.UPDATE, account_options);
                 expect(JSON.parse(res.stdout).error.code).toBe(ManageCLIError.InvalidAccountName.code);
             });
+
+            it('cli account add - cli update account with supplemental groups)', async function() {
+                const { name } = defaults;
+                const supplemental_groups = '303,211';
+                const expected_groups = [303, 211];
+                const account_options = { config_root, name, supplemental_groups};
+                await exec_manage_cli(type, ACTIONS.UPDATE, account_options);
+                const account = await config_fs.get_account_by_name(name, config_fs_account_options);
+                expect(account.nsfs_account_config.supplemental_groups).toStrictEqual(expected_groups);
+            });
+
+            it('cli account add - cli update account with supplemental groups - single group (string))', async function() {
+                const { name } = defaults;
+                const supplemental_groups = '303';
+                const expected_groups = [303];
+                const account_options = { config_root, name, supplemental_groups};
+                await exec_manage_cli(type, ACTIONS.UPDATE, account_options);
+                const account = await config_fs.get_account_by_name(name, config_fs_account_options);
+                expect(account.nsfs_account_config.supplemental_groups).toStrictEqual(expected_groups);
+            });
+
+            it('cli account add - cli update account with supplemental groups - single group (number))', async function() {
```

Comment:
Same - 
Now you're testing account update, please update the it title in this part.
Why do you need the last bracket?
```suggestion
            it('cli account update - cli update account with supplemental groups - single group (number)', async function() {
```